style.css

```diff
@@ -1,10 +1,10 @@
 .body {
-display: flex; 
-flex-direction: column;
-height: 200%; 
-margin: 0; 
-padding: 0;
-background-color: red;
+  display: flex; 
+  flex-direction: column;
+  height: 200%; 
```

Comment:
Selv når du får stylingen her til at virke (laver `.body` om til `body`), så kommer det her ikke til at virke som du gerne vil. Fordi; 200% af hvad? Jeg citerer:

"Percentage values are always relative to another value. The value that the percentage is relative to can be either a value from the element’s parent or from another property of the element itself. For example, if an element’s width is set to 50%, this means that its computed width will be half of that of its parent"
https://tympanus.net/codrops/css_reference/percentage/

Hvis du gerne vil have body til at fylde hele vinduet, kan du sige `height: 100vh`. vh betyder "view height".
Det kan du læse om her: https://www.w3schools.com/cssref/css_units.asp